torch/jit/_monkeytype_config.py

```diff
@@ -0,0 +1,115 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+# mypy: ignore-errors
+
+from typing import Optional, Iterable, List, Dict
+from collections import defaultdict
+
+_IS_MONKEYTYPE_INSTALLED = True
+try:
+    import monkeytype
+    from monkeytype import trace as monkeytype_trace
+    from monkeytype.db.base import CallTraceStore, CallTraceStoreLogger
+    from monkeytype.config import default_code_filter
+    from monkeytype.tracing import CallTrace, CodeFilter
+except ImportError:
+    _IS_MONKEYTYPE_INSTALLED = False
+    print("Warning: monkeytype is not installed. Please install monkeytype to enable Profile-Directed Typing in TorchScript")
```

Comment:
Nit:

refer them to README file about how to install. Also `MonkeyType` is ambiguous, including its github repo link would help.

Reply:
Agreed and fixed